docs/general/userstyles.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Userstyles (Experimental)
+sidebar_position: 6
+---
+
+Userstyles allow users to customize their online experience. Whether you’re looking to change the appearance of a website or automate repetitive tasks, styles offer a world of possibilities.
+
+AdGuard has the option to upload or create your own userstyles. This is an advanced feature, so you will need some knowledge of HTML and CSS.
+
+:::info Supported apps
+
+Currently, two AdGuard apps allow you to create and manage userstyles: AdGuard for Windows and AdGuard for Mac.
```

Comment:
We need to specify versions where we have this functionality.

AG for Mac v2.16 or newer, AG for Windows v7.19 or newer. Planned for AdGuard for Android v4.8.